### DISCUSSION TITLE
`memray` is available on conda-forge channel

### DISCUSSION THREAD

**Discussion by sugatoray** [bloomberg/memray#55] [Announcements]
`memray` is now available on conda-forge channel thanks to `@matthiasdiener`.

- conda-forge `memray` feedstock repo: https://github.com/conda-forge/memray-feedstock

Install with conda:

```sh
conda install -c conda-forge memray
```

[![image](https://user-images.githubusercontent.com/10201242/164844293-6a021a9d-2497-4b31-b3ff-6752c2fdbe9e.png)](https://anaconda.org/conda-forge/memray)

**Comment by pablogsal** [comment]
Thanks a lot @matthiasdiener :heart:
